discovery_engine/lib/src/ffi/types/engine.dart

```diff
@@ -109,6 +109,21 @@ class DiscoveryEngineFfi implements Engine {
     return resultVoidStringFfiAdapter.consumeNative(result);
   }
 
+  /// Sets the excluded sources.
+  @override
+  Future<void> setExcludedSources(
+    Set<String> sources,
+  ) async {
+    // TODO: implement setExcludedSources
```

Comment:
will this be a part of this pr or will we do it in a separate one?

Reply:
this will be a part of a different PR [ref issue](https://xainag.atlassian.net/browse/TY-2456)